fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/debug/DebuggerWithSymbolTables.java

```diff
@@ -269,6 +270,16 @@ private <T> Optional<T> getSilently(@Nonnull final String actionName, @Nonnull f
         }
     }
 
+    @Nonnull
+    public static DebuggerWithSymbolTables sane() {
+        return new DebuggerWithSymbolTables(true);
+    }
+
+    @Nonnull
+    public static DebuggerWithSymbolTables withSanityChecks() {
```

Comment:
These names are a little confusing to me. I take it the idea is that the `sane()` one assumes that all sanity checks would pass, but it may be a little clearer if these were just `withSanityChecks` and `withoutSanityChecks`.

Reply:
Done.